src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Consider using Random object as instance variable. Also, you can create static variable for Colors.values().length. 
This approach helps to avoid creating Random object and getting the length of Color array each time when the getRandomColor() is calling.